README.md

```diff
@@ -1,2 +1,186 @@
-# Leaderboard
-This is a Leaderboard project
+<a name="readme-top"></a>
+
+<div align="center">
+
+<!-- MAIN HEADING -->
+
+  <h3><b>LeaderBoard Project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- INTRO -->
+# 📖 Portfolio <a name="about-project"></a>
+
```

Comment:
- [OPTIONAL] In line with what the previous reviewer mentioned. The title of this project should be Leaderboard and not Portfolio. Please change it to make your README look professional. 👍

Reply:
Thanks